test/e2e/network/network_policy.go

```diff
@@ -353,6 +401,170 @@ var _ = SIGDescribe("NetworkPolicy", func() {
 				testCannotConnect(f, f.Namespace, clientPodName, service, 81)
 			})
 		})
+
+		ginkgo.It("should enforce updated policy [Feature:NetworkPolicy]", func() {
+			ginkgo.By("Creating a network policy for the Service which allows traffic from client-a at port 80")
+			policy := &networkingv1.NetworkPolicy{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "allow-ingress",
+				},
+				Spec: networkingv1.NetworkPolicySpec{
+					// Apply to server
+					PodSelector: metav1.LabelSelector{
+						MatchLabels: map[string]string{
+							"pod-name": podServer.Name,
+						},
+					},
+					// Allow traffic only to one port.
+					Ingress: []networkingv1.NetworkPolicyIngressRule{{
+						From: []networkingv1.NetworkPolicyPeer{{
+							PodSelector: &metav1.LabelSelector{
+								MatchLabels: map[string]string{
+									"pod-name": "client-a",
+								},
+							},
+						}},
+						Ports: []networkingv1.NetworkPolicyPort{{
+							Port: &intstr.IntOrString{IntVal: 80},
```

Comment:
This is nitpicky, but I would prefer variables over magic numbers (EG clientAAllowedPort = 80). Makes the code easier to read without cross-referencing. Not sure if this follows or conflicts with official style though?

Reply:
I'm not sure if there is any official style for this case, I couldn't find any. If there is some I would really appreciate if you could point me to it.
I also think is better to have variables, I was trying to follow the current style for Network Policies tests.
I took a look on some others network tests, I could find some that do not use ports as magic numbers, such as [kubeproxy](https://github.com/MaysaMacedo/kubernetes/blob/add-more-e2e-np-tests/test/e2e/network/kube_proxy.go#L44) and  [networking](https://github.com/MaysaMacedo/kubernetes/blob/add-more-e2e-np-tests/test/e2e/network/networking.go#L109).

Reply:
If you don't mind! I won't block the PR based on this, it's just a recommendation.

Unfortunately, outside of the API, there aren't very strong style guidelines that I know of.

Reply:
Just updated. Thanks!